main/test/ca/uwaterloo/flix/language/phase/TestResolver.scala

```diff
@@ -154,7 +154,7 @@ class TestResolver extends AnyFunSuite with TestUtils {
          |mod B {
          |    def g(): A.S = ???
          |}
-         |"""
+         |""".stripMargin
```

Comment:
Unrelated?

Reply:
How did this test pass before?

Reply:
Resilient parsing? :))))

Reply:
And now it fails to parse? :O

Reply:
It did in a previous iteration where I used `|}` as a delimiter